pyinfra/modules/iptables.py

```diff
@@ -0,0 +1,65 @@
+# pyinfra
+# File: pyinfra/modules/iptables.py
+# Desc: manage iptables rules
+
+'''
+The iptables modules handles iptables rules
+'''
+
+from __future__ import unicode_literals
+
+from pyinfra.api import operation
+
+
+@operation
+def forward(state, host, dport, to_ip, to_port=None,
```

Comment:
Now that this basically handles any rules on the `nat` table I think we can rename this operation to `iptables.nat`.